### DISCUSSION TITLE
Action: Negative testing - Is there a "Not Contains" action?

### DISCUSSION THREAD

**Discussion by MrMPorter** [thunderclient/thunder-client-support#720] [Q&A]
I am looking for a way to test that a string is not in the Response Body. 
For example, I have a route that has options for returning a full set of fields or a partial set of fields.
I can easily test the full set of fields by using Contains.  
        "ResponseBody Contains  'UnitNumber':"
When I use the partial set of fields I want to test that one of the full fields is not in the Response Body. 
        "ResponseBody NotContains  'UnitNumber':"
Suggestions?

**Comment by rangav** [comment]
Thanks @MrMPorter for the feedback. It's a good suggestion.

can you please create a new feature request for this
